venoge/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+add_library(venoge "")
+
+target_sources(venoge
+    PUBLIC
+    PRIVATE
+        "${CMAKE_CURRENT_LIST_DIR}/src/venoge.cpp"
+)
+
+# Define public and private headers
+target_include_directories(venoge
+    PUBLIC
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+        $<INSTALL_INTERFACE:include>
+    PRIVATE 
+        src
+)
+
+# Link libraries: Eigen is only used to test the setup now, not exposed in the API
+target_link_libraries(venoge PRIVATE Eigen3::Eigen)
+
+# `make install` / `cmake --build . --target install`
```

Comment:
Maybe would be better to have the build/install/etc. documented in the README?

Reply:
Indeed. Maybe the readme should be merged first from the other PR?

Reply:
Good point, but that will also come quite naturally, as the README is not finished yet.